packages/main/src/themes/Option.css

```diff
@@ -0,0 +1,6 @@
+.ui5-li-icon {
+	color: var(--sapList_TextColor);
+	min-width: var(--_ui5_list_item_icon_size);
```

Comment:
maybe we can extract icon related styles to ListItemIcon.css similar to the InputIcon.css in order to reuse where it's used. I saw that there are also few styles related to the actionable attribute.

Reply:
Good idea, I did the same for the additional text function.